homeassistant/components/alarm_control_panel/strings.json

```diff
@@ -62,5 +62,18 @@
         }
       }
     }
+  },
+  "issues": {
+    "platform_integration_no_support": {
+      "title": "Platform support not supported",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "title": "Platform support removal",
+            "description": "The {platform} platform for the {domain} integration does not support platform setup.\n\nPlease remove it from your config and select **submit** to close this issue."
```

Comment:
We normally suggest to restart Home Assistant after changing the configuration.yaml to fix the issue. I think we should use the same language here. Look at some integration that has imported the config to config flow and raises a repair issue.